spec/Cabin/SingleSeat.vspec

```diff
@@ -76,50 +74,56 @@ Height:
 
 Cushion:
   type: branch
-  description: Cushion (leg support) signals.
+  description: Describes signals related to the cushion/base of the seat.
+  comment: Cushion is here considered as the part of the seat that supports the thighs.
+           Additional cushions (if any) for support of lower legs is not covered by this branch.
 
-Cushion.Height:
-  datatype: uint16
+Cushion.Tilt:
+  datatype: float
   type: actuator
-  min: 0
-  unit: mm
-  description: Height of the seat cushion (leg support), relative to seat. 0 = Lowermost.
+  unit: degrees
+  description: Seat tilt of cushion, 0 degrees if flat, Positive degrees = tilt backward.
 
 Cushion.Length:
   datatype: uint16
   type: actuator
   min: 0
   unit: mm
-  description: Forward length of cushion (leg support), relative to seat. 0 = Rearmost.
+  description: Cushion length adjustment. 0 = Rearmost (Shortest length of cushion).
 
 Lumbar:
   type: branch
   description: Lumbar signals
 
 Lumbar.Inflation:
-  datatype: uint8
+  datatype: float
   type: actuator
+  unit: percent
   min: 0
-  max: 255
-  description: Lumbar support inflation. 0 = Fully deflated. 255 = Fully inflated.
+  max: 100
+  description: Lumbar support inflation (in/out position). 0 = Fully deflated (innermost position).
+               100 = Fully inflated (outermost position).
+  comment: Even if signal name implies that the functionality
 
 Lumbar.Height:
   datatype: uint8
   type: actuator
   min: 0
   unit: mm
-  description: Lumbar support position. 0 = Lowermost.
+  description: Lumbar support height position. 0 = Lowermost.
 
 SideBolster:
   type: branch
-  description: Side bolster settings
+  description: Backrest side bolster (lumbar side support) settings.
 
 SideBolster.Inflation:
   datatype: uint8
   type: actuator
+  unit: percent
   min: 0
-  max: 255
-  description: Side bolster support inflation. 0 = Fully deflated. 255 = Fully inflated.
+  max: 100
+  description: Side bolster support inflation. 0 = Fully deflated (widest lumbar setting, least support).
+               100 = Fully inflated (thinnest lumbar setting, maximum support).
 
 HeadRestraint:
```

Comment:
Could be shorter "HeadRest"

Reply:
Indeed, I can suggest that and see what VSS people say, if they think it is worth the effort to just change the name. But as several other VSS seat signals change name I think that the extra work shall be limited.

Reply:
This point is not so important for me. Was just one finding.
In fact, I prefer having longer names if it make things clearer (less ambiguous).